src/components/PlayerInfo.vue

```diff
@@ -9,23 +9,23 @@
       Range: {{data.range}}, Terraforming cost: {{3 - data.terraformCostDiscount}}o<br/>
       <span v-if="faction === 'Ivits'">Fed value: {{player.fedValue }}, No fed value: {{player.structureValue - player.fedValue }} <br/></span> 
  
-        <span style="white-space: nowrap; line-height: 1em">
+      <span style="white-space: nowrap; line-height: 1em">
         Steps: 
         <span v-for="i in [0, 1, 2, 3]" :key="i" :class="{'ml-2': i > 0}">
-          <span v-for="planet in planetsWithSteps(i)"  :key="planet"  >
-            <svg width="15" height="20" viewbox="0 0 15 15" >
+          <span v-for="planet in planetsWithSteps(i)" :key="planet"  >
+            <svg width="16" height="20" viewbox="0 0 16 15" >
               <circle :cx="8" :cy="8" :r="6"  :class="['player-token', 'planet-fill', planet]" />
             </svg>
           </span> 
           <span>{{i}}</span>
         </span>
       </span><br/>
       <span style="line-height: 1em" v-if="hasPlanets">Colonized: 
-        <span v-for="(count, planet, index) in player.ownedPlanetsCount"  :key="planet"  :class="'ml-2'">
-          <svg width="15" height="20" viewbox="0 0 15 15" >
+        <span v-for="(count, planet, index) in player.ownedPlanetsCount"  :key="planet"  class="'mr-2'">
```

Comment:
You can remove the quotes, `class="mr-2"` instead of `class="'mr-2'"`.

The `:` in vue js is treating the content as a variable, so that means using quotes around stings. Without the `:`, the content is just a string, there's no need to add quotes around it.